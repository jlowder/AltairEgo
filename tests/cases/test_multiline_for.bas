10 FOR I=1 TO 2 : PRINT "I=";I
20 GOSUB 100
30 NEXT I
40 PRINT "DONE"
50 END
100 PRINT "SUBROUTINE"
110 RETURN
