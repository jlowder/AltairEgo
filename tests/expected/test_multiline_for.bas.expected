Altair Ego: Emulating Altair BASIC 32K Rev. 3.2
OK
OK
I= 1
SUBROUTINE
I= 2
SUBROUTINE
DONE
OK
